index.src.html

```diff
@@ -778,6 +1097,15 @@
             requirements, since it might not be possible to synchronize tracks
             from sources.
           </p>
+          <p class="note">
+            This approach is simple to use but comes with the following
+            caveats: it might not be available and the resolutions of two
+            tracks are intended to be the same. The alternative approach is
+            that a web developer implements, following the <a>algorithm to map
+            depth pixels to color pixels</a> and the corresponding <a href=
+            "https://www.w3.org/wiki/Media_Capture_Depth_Stream_Extension#Examples">
```

Comment:
Thanks, removed the part "in WebGL vertex shader." as for recognition use cases, CPU implementation could be valid approach too (e.g. when processing 1080p color frame developer is interested in depth value for the specific pixel and doesn't necessary require full color frame processing).